plenum/test/consensus/test_consensus_dp_batches.py

```diff
@@ -0,0 +1,39 @@
+from plenum.test.pool_transactions.helper import sdk_add_new_nym_without_waiting
+from stp_core.loop.eventually import eventually
+
+from plenum.test.delayers import ppDelay, cDelay, pDelay
+
+from plenum.test.stasher import delay_rules
+
+
+def test_check_cdp_pp_storages(looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_steward):
+    def check_all_empty(replica, reverse=False):
```

Comment:
What is the purpose of reverse here?

Reply:
to check that collection is NOT empty, if reverse is True